app/src/common/components/article/line.jsx

```diff
@@ -0,0 +1,4 @@
+/** Component that displays an article as a line. */
+export function ArticleLine({article}) {
+    return <li>{article.title} {article.description} {article.content}</li>;
```

Comment:
I was under the impression what the displayName wasn't necessary if the function or class already had a name. Anyway this is just a placeholder component that's going to be replaced very soon.